spec/dummy/.gitignore

```diff
@@ -21,3 +21,9 @@
 /public/webpack/*
 
 /spec/examples.txt
+
+#Ignore ReScript build files
+/.merlin
+/lib/bs/
```

Comment:
The whole `lib` should be ignored.

Reply:
@gscarv13 BS files should not be placed in the /lib directory. 

@alexfedoseev what is the bs directory? The lib directory is used by rail, so it should not be ignored.

Reply:
@justin808 The ReScript compiler put the `bs/` inside `lib/`. Looking at the documentation I couldn't find a way to change it